http/src/main/resources/reference.conf

```diff
@@ -347,6 +347,7 @@ azure {
      environment = "dev"
      environment-base = "live"
      instrumentation-enabled = false
+     image: us-central1-docker.pkg.dev/dsp-artifact-registry/mf-dataplane-metircs-wip/wds
```

Comment:
just checking ... is `metrics` spelled correctly?

Reply:
nope :face_palm

Reply:
Well this did exactly what I wanted it to, thanks for flagging the typo, probably saved me quite a bit of time.